### ISSUE TITLE
Share some configuration/data with @buddy

### ISSUE THREAD

**Issue by ben** [zendesk/term-check#32]
The "Buddy" chat bot for Slack over at https://github.com/zendesk/buddhy-botkit has some similar features. It might be worthwhile to see if some data and configuration could be shared.

**Comment by ragurney** [comment]
👋 what kind of configuration did you have in mind? The only stuff I could think of would be the term list this bot uses, though I'm not sure that would be helpful to the buddhy-bot since this checks the context of code, while the bot is more looking for terms said in public slack channels and they might not line up?

**Comment by ben** [comment]
Yeah it would probably be the [term regexes](https://github.com/zendesk/buddhy-botkit/blob/76560a84eb58ee9047da158876e53552545a6d73/scripts/guys.js#L8-L51), and _maybe_ the [suggested alternatives](https://github.com/zendesk/buddhy-botkit/blob/76560a84eb58ee9047da158876e53552545a6d73/scripts/guys.js#L83-L123). You're definitely right that context matters, so this issue is more of a "let's explore this idea" rather than a bug report.
